src/test.js

```diff
@@ -1,9 +1,14 @@
+import * as Comlink from 'comlink';
+// eslint-disable-next-line import/no-webpack-loader-syntax
+import Worker from 'worker-loader!./worker';
 import * as data from './data.js';
 import { global } from './global.js';
 import * as simulator from './simulator.js';
 import * as utils from './utils.js';
 
-export function outputManyRuns(players, selectedDungeon, numRuns) {
+const workerPool = [Comlink.wrap(new Worker()), Comlink.wrap(new Worker()), Comlink.wrap(new Worker()), Comlink.wrap(new Worker())]
```

Comment:
I don't actually know that using multiple workers is having any impact here. Just assumed it was better.